docs/proposal/bypass.md

```diff
@@ -0,0 +1,72 @@
+## Background
+
+Currently, if the mesh service is turned on when communication between servers fails, it is difficult to determine whether the problem is caused by the mesh service taking over the traffic, or whether there is a logic error in the application itself. Therefore, we provide a feature to shut down the mesh service and remove it from the communication traffic path to pinpoint whether the problem is caused by the mesh service or an error in the application itself.
+
+## Bypass Function Design
+
+Bypass is a general mesh function used to remove sidecar/kmesh mesh services from the traffic path. The following two scenes show the traffic path after the bypass function is enabled.
+
+### Scene 1: sidecar
+
+Among these two figures, the mesh service is enabled in the cluster to take over the traffic. The first figure shows the traffic path before the bypass function is enabled. We can see from it that the traffic path is server A -> sidecar -> sidecar - > server B, when the communication between server A and server B fails, from the black box level, the user cannot determine whether there is a problem with the sidecar or the server itself. In Figure 2, the bypass function of pod1 is enabled, so that the sidecar in pod1 is removed from the traffic path. If server A and server B can communicate normally at this time, it proves that there is a problem with the sidecar.
```

Comment:
There is another case: kmesh shortcut the server side

Reply:
> There is another case: kmesh shortcut the server side
Yes, I've added the relevant logic.